src/config/mapping.json

```diff
@@ -1,6 +1,6 @@
 {
   "labels": {
-    "https://bitcointalk.org": "Bitcointalk",
+    "https://bitcointalk.org/": "Bitcointalk",
```

Comment:
Any reason why we added the root path to this particular label?

Reply:
I find that it is not consistent throughout the file that we add the paths to the url

Reply:
yeah, we need to have a perfect match with what is being used as the domain on ES_Search.
Passing `https://bitcointalk.org` instead of `https://bitcointalk.org/` to ES_Search will not give any results because there's no `https://bitcointalk.org` in the ES_index.

basically how the domain appears on the index is our source of truth